code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -79,6 +79,7 @@
 	new /obj/item/weapon/storage/box/flashbangs(src)
 	new /obj/item/weapon/shield/riot/tele(src)
 	new /obj/item/weapon/storage/belt/security/full(src)
+	new /obj/item/weapon/tank/jetpack/oxygen/security(src)

```

Comment:
That will conflict with my other PR that places a oxygen tank in that spot, and I'm being consistant with other jetpack placements. Hence why I'm putting it in his locker.